debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresChangeEventSourceCoordinator.java

```diff
@@ -47,9 +49,10 @@ public PostgresChangeEventSourceCoordinator(Offsets<PostgresPartition, PostgresO
                                                 EventDispatcher<PostgresPartition, ?> eventDispatcher, DatabaseSchema<?> schema,
                                                 SnapshotterService snapshotterService, SlotState slotInfo,
                                                 SignalProcessor<PostgresPartition, PostgresOffsetContext> signalProcessor,
-                                                NotificationService<PostgresPartition, PostgresOffsetContext> notificationService) {
+                                                NotificationService<PostgresPartition, PostgresOffsetContext> notificationService, BeanRegistry beanRegistry,
```

Comment:
I must have missed this in earlier changes in the PR, but I noticed here we pass both the BeanRegistry and the ServiceRegistry. Would you agree its satisfactory to pass the ServiceRegistry and only then get the bean registry where needed to keep the signature shorter and "more clean" than it currently is? :grin:

Reply:
Regarding that, we can continue the discussion in the other comment.